en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md

```diff
@@ -0,0 +1,122 @@
+# ISO8583 Connector Example
+
+Given below is a sample scenario that demonstrates how to WSO2 ISO8583 connector send an ISO8583 message to the financial networks using WSO2 micro integrator.
+
+## What you'll build
+
+There are many such standards exist. The ISO8583 message format is used for financial transaction, such as ATM, POS, Credit Card, Mobile Banking, Internet Banking, KIOSK, e-commerce etc. transactions.
+The financial transaction is communication between two systems through a socket connection. After connection established, each system can send messages in ISO 8583 format which commonly will be requested and the other system will response. 
+
+For an example the purchase made in a store may travel from the merchant terminal through another require a network or networks to the issuing bank where the card holders account is held.
+Cardholder-originated transactions include purchase, withdrawal, deposit, refund, reversal, balance inquiry, payments and inter-account transfers. ISO 8583 also defines system-to-system messages for secure key exchanges, reconciliation of totals, and other administrative purposes.The response or authorizing or decline the transaction needs to be returned by the same route to the terminal.
+
+Given below is a sample API that illustrates how you can configure with the `init` operation and then use the `iso8583.sendMessage` operation for the send an ISO8583 message to the financial transactions.
+
+**initialize the connector**
+````xml
+<iso8583.init>
+       <serverHost>localhost</serverHost>
+       <serverPort>5010</serverPort>
+</iso8583.init>
+````
+**Properties**
+
+| Property        | Description |
+| ------------- |-------------|
+| serverHost   | The host of the server. |
+| serverPort   | The port on which the server can start listening for messages. |
+
+**Sending an ISO8583 message**
+
+To send an ISO8583 message, you need to use the <iso8583.sendMessage/> operation in your configuration. You can use a REST client to send an XML message, set the Content-Type header in the REST client as application/xml, and then POST the body in XML format.
+
+The following diagram illustrates how to send an ISO8583 message using a REST client.
+
+<img src="/assets/img/connectors/ISO8583-connector.png" title="ISO8583 Connector" width="800" alt="ISO8583 Connector"/>
```

Comment:
Shall we add some description of what exactly is happening in the diagram?

Reply:
Added.